app/docs/setup_code.tsx

```diff
@@ -53,11 +55,12 @@ export default class SetupCodeComponent extends React.Component<Props, State> {
 
   componentDidUpdate(prevProps: Props) {
     if (this.props.bazelConfigResponse !== prevProps.bazelConfigResponse) {
-      this.setConfigResponse(this.props.bazelConfigResponse);
+      this.setConfigResponse(this.props.bazelConfigResponse!);
```

Comment:
nit: since `this.props.bazelConfigResponse` is optional (based on the `Props` type) we might want to keep `setConfigResponse()` supporting undefined response values (`?:`) and drop the `!` assertion here (`fetchAPIKeyValue` probably needs a small tweak as well)

Reply:
done